pagopa/pn-frontend
# PN-13437 - Rework PecContactItem

## Short description
Refactor the `PecContactItem` component to simplify its logic by removing certain functionalities, such as the ability to delete or add PEC addresses when SERCQ is enabled. Additionally, update the graphical interface by eliminating the `DigitalContactsCard` and removing text in the `PecValidationItem`.

## List of changes proposed in this pull request
- Rework `PecContactItem`
- Rework `PecValidationItem`
- Remove `PecValueDialog` (component and tests)
- Remove useless translations

## How to test
Start PF/PG and verify the functionality of the `PecContactItem`. Ensure the following features work as expected:
- Adding a PEC
- Editing an existing PEC
- Deleting PEC validation
